### PR TITLE
[device config] Adding configuration for default route fallback

### PR DESCRIPTION
**Pull request by Ndancejic**
#### Why I did it:

Set sai_tunnel_underlay_route_mode attribute to fallback to default route if more specific route is unavailable. 

#### How I did it:

Added the configuration option: sai_tunnel_underlay_route_mode=1 in
  device\arista\x86_64-arista_7050cx3_32s\
  Arista-7050CX3-32S-C32\td3-a7050cx3-32s-32x100G.config.bcm

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012
- [ ] 202106
- [ ] 202111

#### Description for the changelog
Adding configuration for default route fallback

### CODE REVIEW

**Comment by Ndancejic** [comment]
Tests are failing because of this error message:
`[line 8] Error: sai_tunnel_underlay_route_mode is not permitted`

**Comment by prsunny** [comment]
Please add the attribute to permitted list

File: device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/td3-a7050cx3-32s-32x100G.config.bcm [hunk 1]
```diff
@@ -5,6 +5,7 @@ host_as_route_disable=1
 use_all_splithorizon_groups=1
 riot_enable=1
 sai_tunnel_support=1
+sai_tunnel_underlay_route_mode=1
```
Comment (prsunny) [inline device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/td3-a7050cx3-32s-32x100G.config.bcm:8]: Please add the attribute to other relevant config files as well

**Review by prsunny — APPROVED** [review]

**Comment by qiluo-msft** [comment]
This commit could not be cleanly cherry-picked to 202012. Please submit another PR.

**Comment by prsunny** [comment]
@Ndancejic , can you please raise a separate PR for 202012?

**Pull request merged**